### PR TITLE
promisify and lipstickify

### PR DESCRIPTION
**Pull request by SamVerschueren**
the usual stuff 🐑 + 💄

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -27,10 +27,21 @@ console.log(unixPathMax.sync());
...
+### unixPathMax.sync()
+
+Returns the maximum permitted length of a path.
+
+
 ## CLI
 
 ```
-$ npm install --global unix-path-max
+$ npm install --g unix-path-max
```
Comment (sindresorhus) [inline readme.md]: ?

Comment (SamVerschueren) [inline readme.md]: Oeps :). In bed right now, fix it tomorrow.

**Comment by SamVerschueren** [comment]
fixed

**Comment by sindresorhus** [comment]
leo-yay

**Pull request merged**